pkg/openstack/loadbalancer.go

```diff
@@ -785,6 +804,20 @@ func applyNodeSecurityGroupIDForLB(network *gophercloud.ServiceClient, svcConf *
 			if mc.ObserveRequest(res.Err) != nil {
 				return fmt.Errorf("failed to update security group for port %s: %v", port.ID, res.Err)
 			}
+
+			tags := getCustomLoadBalancerTags(service, svcConf)
+
+			mc = metrics.NewMetricContext("security_group_tag", "replace")
+			_, err := neutrontags.ReplaceAll(network, "security_groups", port.ID, neutrontags.ReplaceAllOpts{Tags: tags}).Extract()
+			if mc.ObserveRequest(err) != nil {
+				return fmt.Errorf("failed to add tag %s to port %s: %v", tags, port.ID, err)
+			}
+
+			mc = metrics.NewMetricContext("floating_ip_tag", "replace")
+			_, err = neutrontags.ReplaceAll(network, "floatingips", port.ID, neutrontags.ReplaceAllOpts{Tags: tags}).Extract()
+			if mc.ObserveRequest(err) != nil {
+				return fmt.Errorf("failed to add tag %s to port %s of floating_ips: %v", tags, port.ID, err)
+			}
```

Comment:
This shouldn't be here, this function doesn't have anything to do with floating IPs.

Also this won't ever work, you specify `port.ID` which will not match a floating IP.